nginx.tpl.conf

```
@@ -33,17 +33,37 @@ http {
     gzip_proxied any;
     gzip_types *;
 
+    map $cookie_lang $lang {
+        default 'en';
+        zh      zh;
+        ko      ko;
+    }
+
     server {
         listen 80 default_server;
 
         error_page   404  /404.html;
 
         rewrite ^/assets/white_paper.* https://github.com/jibrelnetwork/white_paper permanent;
 
+        location = / {
+            return 302 $scheme://$http_host/$lang/;
```

```suggestion
            return 302 $scheme://$host/$lang/;
```
In the downstream container, you will always have a port number in this variable. You shouldn't use it in redirects.